File: Arcade/Intro/The Journey Begins/CPlusPlus/checkPalindrome.cpp
bool checkPalindrome(std::string inputString)
{

    int start, end;

    start = 0;
    end = inputString.size() - 1;

    while (start < end)
    {

        if (inputString[start++] != inputString[end--])
        {
            return false;
        }
    }
    return true;
}

File: Arcade/Intro/The Journey Begins/JavaScript/checkPalindrome.js
function checkPalindrome(inputString) {
  var start, end;

  start = 0;
  end = inputString.length - 1;

  while (start < end) {
    if (inputString[start++] != inputString[end--]) {
      return false;
    }
  }
  return true;
}
